wiki/osu!supporter/en.md

```diff
@@ -1,39 +1,35 @@
 # osu!supporter
 
-![osu!supporter tag](/wiki/shared/osu!supporter.png "osu!supporter tag")
+![osu!supporter link on main page](./img/sup.png "osu!supporter link on main page.")
 
-![osu!supporter link on main page](Osu-support.png " osu!supporter link on main page.")
-
-Refer to link: <https://osu.ppy.sh/p/support>
-
-Laconic: Donation is used to sustain osu! overall
+![osu!supporter tag](/wiki/shared/osu!supporter.png "osu!supporter tag") [Support osu!](https://osu.ppy.sh/p/support)
```

Comment:
just pointing this out, but it will look like this now:
![image](https://user-images.githubusercontent.com/14264992/30526412-abe52bfc-9bdf-11e7-8602-2a881afeba1f.png)
left = GitHub
right = osu!web